gbpservice/nfp/configurator/lib/constants.py

```diff
@@ -31,6 +31,7 @@
 VPN = 'vpn'
 VYOS = 'vyos'
 HAPROXY = 'haproxy'
+PALOALTO = 'paloalto'
```

Comment:
Lets not add enterprise related constants into this file. Move this to vendor specific constants file.